config.toml

```diff
@@ -5,6 +5,7 @@ PDFS_DIR = "pdfs"
 PROJECTS_DIR = "projects"
 LOGS_DIR = "logs"
 REPOS_DIR = "repos"
+MODEL_PATH = "<YOUR_MODEL_PATH>"
```

Comment:
What about renaming this one to "LLAMA_MODEL_PATH" or similar? Maybe in the future we would need to add other kind of local models, like Mixtral or phi, so its better to have explicit names

Reply:
Agreed! Made those changes. Thanks for sharing!